pixi.toml

```diff
@@ -13,15 +13,19 @@ documentation = "https://deltares.github.io/Ribasim/"
 repository = "https://github.com/Deltares/Ribasim"
 
 [tasks]
+install-ribasim-python ={ path = "python/ribasim", editable = true }
+install-ribasim-api ={ path = "python/ribasim_api", editable = true }
+install-ribasim-testmodels ={ path = "python/ribasim_testmodels", editable = true }
```

Comment:
With editable installs the installation is no longer done with a task, but automatically if listed under pypi dependencies. See https://github.com/Deltares/Ribasim-NL/pull/75 for an example.